pinot-segment-local/src/main/java/org/apache/pinot/segment/local/utils/TableConfigUtils.java

```diff
@@ -378,12 +386,75 @@ public static void validateIngestionConfig(TableConfig tableConfig, @Nullable Sc
 
           FunctionContext functionContext = expressionContext.getFunction();
           validateIngestionAggregation(functionContext.getFunctionName());
-          Preconditions.checkState(functionContext.getArguments().size() == 1,
-              "aggregation function can only have one argument: %s", aggregationConfig);
+
+          List<ExpressionContext> arguments = functionContext.getArguments();
+
+          if (("distinctcounthll".equals(functionContext.getFunctionName()))
```

Comment:
We need to use canonical name (removing underscore). Currently if the function name is `distinct_count_hll` or `sum_precision` it will fail